scene/gui/text_edit.cpp

```diff
@@ -2420,26 +2420,44 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 
 						//simple unindent
 						int cc = cursor.column;
+
+						const int len = text[cursor.line].length();
+						const String &line = text[cursor.line];
+
+						int left;
```

Comment:
Left is not always defined causing crashes and weird behaviour. Should be initialised to 0

Reply:
Indeed, missed that completely.